### ISSUE TITLE
Consistent application of state-dependant pool/group/collection name

### ISSUE THREAD

**Issue by SignpostMarv** [SignpostMarv/Satisfactory-Production-Calculator-UI#10]
Where the state-dependent appending of pool/group id is added to the title attribute, change text contents to non-state-dependent name.

i.e. if title/aria-label says "Foo (21)", visible text should be "Foo"
